llvm/lib/Transforms/Scalar/LoopInterchange.cpp

```diff
@@ -106,7 +107,10 @@ static cl::list<RuleTy> Profitabilities(
                clEnumValN(RuleTy::PerInstrOrderCost, "instorder",
                           "Prioritize the IVs order of each instruction"),
                clEnumValN(RuleTy::ForVectorization, "vectorize",
-                          "Prioritize vectorization")));
+                          "Prioritize vectorization"),
+               clEnumValN(RuleTy::Ignore, "ignore",
+                          "Ignore profitability, force interchange (does not "
+                          "work with other options)")));
```

Comment:
```suggestion
                          "Ignore profitability, force interchange")));
```

Or do you want to reject cases where `ignore` is specified with other options?

Reply:
Yes, exactly. That's why I wrote this comment:

    // We ignore the force option when it appears in a list, i.e. it
    // should occur as the only option to be effective, as mentioned in the
    // help.

This is a complication of moving this to an existing option that accepts a list of options. We have to define the behaviour of "option-a, optionb, ..., ignore" and I think dropping and ignoring it is the best.

Reply:
Done, left a debug message.